.github/workflows/ci.yml

```diff
@@ -82,7 +82,7 @@ jobs:
           function _make() { make -j$(nproc) -s O=out $*; }
           function download_plug_and_trust() { mkdir -p $HOME/se050 && git clone --single-branch -b v0.4.2 https://github.com/foundriesio/plug-and-trust $HOME/se050/plug-and-trust || (rm -rf $HOME/se050 ; echo Nervermind); }
 
-          function download_scp_firmware() { git clone --single-branch https://github.com/ARM-software/SCP-firmware.git $HOME/scp-firmware || echo Nervermind; }
+          function download_scp_firmware() { { git clone --single-branch https://github.com/ARM-software/SCP-firmware.git $HOME/scp-firmware && git -C $HOME/scp-firmware checkout v2.13.0; } || echo Nervermind; }
```

Comment:
I think we could add `-n`/`--no-checkout` option to the `git clone` command and `-q`/`--quiet` to the `git checkout command`.

Reply:
Yes, and yes.

Reply:
I prefer the short forms `-n/-q` to reduce the line size. Tell me if you prefer the long arguments for everyone to easier understand the command line.

Reply:
`-n/-q` is fine IMO.